### PR TITLE
Subscription

### PR DESCRIPTION
**Pull request by Dragonfly911117**
### Fixes
- #32 
- #54 

#### GET
- `/api/subscription`: get subscription relations with either `account_uuid` or `shop_uuid`
#### POST
- `/api/subscription`: Add subscription relations with token `shop_uuid`(Admins may force other users to sub some shop)
#### DELETE
- `/api/subscription`: Remove subscription relations with token `shop_uuid`(Admins may force other users to unsub some shop)
### Misc Changes
- init subscription tuples

### CODE REVIEW

File: app/router/account.py [hunk 1]
```diff
@@ -177,5 +178,5 @@ async def update_account(
     except ValueError as e:
         raise HTTPException(
             status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
-            detail=ErrorModel(msg=str(e)).model_dump()
+            detail=str(e)
```
Comment (Vghxv) [inline app/router/account.py:181]: consider adding meaningful explanation in the context of the api?

**Review by Vghxv — APPROVED** [review]
lgtm, not good to frontend

File: app/router/subscription.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    },
+)
+async def subscribe(
+        account: Annotated[
+            Account,
+            Depends(auth.get_current_active_user)],
+        shop_uuid: str,
+        account_uuid: str | None = None
+):
+    if not (account.role == 1 and account_uuid):
```
Comment (garry911128) [inline app/router/subscription.py:78]: 用戶可以訂閱自己是可以的嗎

Comment (Vghxv) [reply app/router/subscription.py:78]: nno

**Pull request merged**